Documentation/codereview-policy.md

```diff
@@ -0,0 +1,45 @@
+### Code Review Policy
+
+#### How to review pull requests  
+
+This is a guide for core developers and contributors who want to help with code reviewing PRs on the fullnode.  
+We welcome anyone to help with code reviews, from typos to suggestions in design.   
+
+#### Why are code reviews important?  
+It's essential that PRs are properly code-reviewed.   
+Changes to the code, even small changes as incrementing an integer, can result in critical consensus bugs and loss of funds if not reviewed properly.  
+Some PRs are more severe than others in importance (for example changes to tests may not require the same level of checks as changes to the consensus code).  
```

Comment:
I don’t like this part about devaluing the importance of tests. I would suggest comparing something else with the importance of consensus code. Ie pick another area to compare as being less important, eg wallet button repositioning

Reply:
Is not devaluing the importance of tests. The tests are important, but no one will ever lose money just because the test is broken in a way that it gives false-positive result. And if it is broken in a way that it gives false-negative then it will just prevent PR being merged. So basically you cannot make a mistake in a test that will immediately break something for the users. So obviously the test code is less critical.

Reply:
What I meant was....   Perhaps this would be better...

Some PRs are more severe than others in importance (for example changes to consensus code need a high level of checks).

I don't see any need to compare prod code to tests.